actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -52,5 +52,11 @@ class Railtie < Rails::Railtie # :nodoc:
 
       ActionDispatch.test_app = app
     end
+
+    initializer "action_dispatch.server_timing" do
+      if default_events = config.action_dispatch.delete(:server_timing)
+        config.app.middleware_use ActionDispatch::Middleware::ServerTiming, default_events
```

Comment:
We probably want to insert this middleware in a  specific location in the stack, just after the `ActionDispatch::Executor` should be enough I think